cli/runExpoAppPublish.sh

```diff
@@ -306,10 +321,9 @@ function main() {
   # Update the app.json with build context information - Also ensure we always have a unique IOS build number
   # filter out the credentials used for the build process
   jq --slurpfile envConfig "${CONFIG_FILE}" \
-    --arg RELEASE_CHANNEL "${RELEASE_CHANNEL}" \
     --arg BUILD_REFERENCE "${BUILD_REFERENCE}" \
     --arg BUILD_NUMBER "${BUILD_NUMBER}" \
-    '.expo.releaseChannel=$RELEASE_CHANNEL | .expo.extra.build_reference=$BUILD_REFERENCE | .expo.ios.buildNumber=$BUILD_NUMBER | .expo.extra=.expo.extra + $envConfig[]["AppConfig"]' <  "./app.json" > "${tmpdir}/environment-app.json"
```

Comment:
Any reason why release channel was removed? It does warn about it but some of the apps we have deployed rely on it.

Reply:
Only to remove the warning. If its not part of the app.json schema, then its not going to used for anything as far as Expo is concerned. How do apps end up using it?

Reply:
It's still required, expo inject it as part of their own build process and inject it just before they publish the OTA. This is why its not pat of the expected values from the user so they give you a warning about it

Reply:
ok - I can revert that change. I need to add setting of the wait time for OTA as well as Luke tells me setting it to zero (as is currently the default) basically causes the app to bypass OTA checking.

Reply:
I've reverted the code to include the release channel, and added the WaitMs parameter to trigger an OTA check on startup.